homeassistant/components/litterrobot/config_flow.py

```diff
@@ -0,0 +1,61 @@
+"""Config flow for Litter-Robot integration."""
+import logging
+
+from pylitterbot.exceptions import LitterRobotException, LitterRobotLoginException
+import voluptuous as vol
+
+from homeassistant import config_entries, core
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+
+from .const import DOMAIN  # pylint:disable=unused-import
+from .hub import LitterRobotHub
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {vol.Required(CONF_USERNAME): str, vol.Required(CONF_PASSWORD): str}
+)
+
+
+async def validate_input(hass: core.HomeAssistant, data: dict):
```

Comment:
We can likely do away with this and collapse it below as it doesn't save you any lines and returns the same data that was passed in